lib/commands/simctl.js

```diff
@@ -0,0 +1,71 @@
+import { errors } from 'appium/driver';
+
+/**
+ * List of subcommands which accept 'device' target.
+ */
+const SUBCOMMANDS_HAS_DEVICE = [
+  'addmedia',
```

Comment:
lets maybe exclude addMedia and push commands from the list as well as they allow to read any file on the server and anyway have external wrappers, that verify file permissions

Reply:
on the second thought we could also keep them as simulator apps anyway have access to the whole host FS

Reply:
Sounds good to exclude addmedia. We provide it as regular command so we don't need to provide it as this PR